serializability-checker-compiler-plugin/src/test/resources/GenericsTest3.scala

```diff
@@ -0,0 +1,18 @@
+package org.random.project
+
+import akka.actor.typed.ActorSystem
+import akka.grpc.scaladsl.WebHandler
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse}
+
+import scala.concurrent.Future
+
+object GenericsTest3 {
+
+  def invokeGrpcWebHandler(handlers: List[PartialFunction[HttpRequest, Future[HttpResponse]]])
+                                  (implicit as: ActorSystem[_]): HttpRequest => Future[HttpResponse] = {
+    // For some reason type parameter on below method call for implicitly passed ActorSystem[_] is not `scala.Any`
```

Comment:
Wow... did you manage to reproduce it on any other type? no need to do an extra research ofc, just curious

Reply:
(other than `WebHandler`)

Reply:
No, I tried many times without using GRPC on some custom code with self-made ActorSystem[Any] but failed to reproduce the `._$1` dollar-digit style compiler type naming :/

Reply:
That's why I decided to use GRPC - didn't find any other way to reproduce this case

Reply:
Wooow so the Łukasz's issue is even more exotic than I expected ;p

Reply:
Might even be a bug/glitch/inconsistency in Scala (compiler) itself